README.md

```diff
@@ -1,2 +1,395 @@
+            check_3["✅"]
+        end
+    end
+
+style alpine fill:#ffffff,stroke:#444444;
+style check_1 fill:#D1FAE5,stroke:#D1FAE5;
+style check_2 fill:#D1FAE5,stroke:#D1FAE5;
+style check_3 fill:#D1FAE5,stroke:#D1FAE5;
+style alpine fill:#ffffff,stroke:#444444;
+```
+
+_Diagram showing common benefits of Alpine and HPC clusters._
+
+Alpine is a [High Performance Compute (HPC) cluster](https://en.wikipedia.org/wiki/High-performance_computing).
+HPC environments provide shared computer hardware resources like [memory](https://en.wikipedia.org/wiki/Computer_memory), [CPU](https://en.wikipedia.org/wiki/Central_processing_unit), [GPU](https://en.wikipedia.org/wiki/Graphics_processing_unit) or others to run performance-intensive work.
+Reasons for using Alpine might include:
+
+- __Compute resources:__ Leveraging otherwise cost-prohibitive amounts of memory, CPU, GPU, etc. for processing data.
+- __Long-running jobs:__ Completing long-running processes which may take hours or days to complete.
+- __Collaborations:__ Sharing a single implementation environment for reproducibility within a group (avoiding "works on my machine" inconsistency issues).
+
+### How does Alpine work?
+
+```mermaid
+flowchart LR
+    users["👥 Users"]
+    subgraph alpine["🖥️ Alpine"]
+        slurm["🗓️ Slurm"]
+        subgraph login_nodes["🔑 Login node(s)"]
+            acompile["📄 acompile"]
+            slurm_cmd["⌨️ Slurm cmd's"]
+        end
+        subgraph compute_nodes["⚙️ Compute node(s)"]
+            hardware["⚙️ Compute\nResources"]
+            modules["💿 Software via\nmodules pkg"]
+        end
+
+        subgraph storage["📂 Storage"]
+            local_storage["📁 Local Storage\n(sometimes temporary)"]
+        end
+    end
+    remote_storage["📁 External Storage\n(user specified / configured)"]
+
+    users --> | preconfigured\nSlurm access| acompile --> slurm
+    users --> | direct access | slurm_cmd --> slurm
+    slurm --> |"schedules\n(shared) use of"| hardware
+    slurm --> | provides\naccess to| modules --> | which may\n leverage| hardware
+    hardware --> | may deliver\nresults to| local_storage
+    hardware --> | or deliver\nresults to| remote_storage
+
+style alpine fill:#ffffff,stroke:#444444;
+style slurm fill:#F0F9FF,stroke:#075985;
+```
+
+_Diagram showing high-level user workflow and Alpine components._
+
+Alpine's compute resources are used through compute nodes in a system called [Slurm](https://github.com/SchedMD/slurm).
+Slurm helps coordinate shared and configurable access to the compute resources.
+Data for or from Slurm work may be stored temporarily on local storage or on user-specific external (remote) storage.
+
+> ℹ️ __Wait, what are "nodes"?__
+>
+> A simplified way to understand the architecture of Slurm on Alpine is through login and compute "nodes" (computers).
+> Login nodes act as a way to prepare and submit processes which will be completed on compute nodes.
```

Comment:
You might want to call back to the definition of a job from before, e.g.:

```suggestion
> Login nodes act as a place to prepare and submit jobs which will be completed on compute nodes. Login nodes are never used to execute Slurm jobs, whereas compute nodes are exclusively accessed via a job.
```

Reply:
Thanks - way clearer! Committing this in.